lib/mat/mat-rest-commons/0.0.1-SNAPSHOT/maven-metadata-local.xml

```diff
@@ -7,17 +7,17 @@
     <snapshot>
       <localCopy>true</localCopy>
     </snapshot>
-    <lastUpdated>20200305190420</lastUpdated>
```

Comment:
we should .gitignore these files in lib/mat IMO. We shouldn't be checking these in every time.

Reply:
@cday-semanticbits - that will affect how the maven repo works. It's a workaround for the dependencies. But there is another PR where I moved 'install-plugin' to a dedicated profile, so we don't update the repo files every time.